### DISCUSSION TITLE
Is there any way to change the defaults of the script when it runs?

### DISCUSSION THREAD

**Discussion by SebSaga** [thygate/stable-diffusion-webui-depthmap-script#88] [Q&A]
If it's possible I would love to change the default state of the "Combine into one image." checkbox so that it is unchecked on launch.

Not a huge deal, but figured I'd ask if it isn't super hard for me to update any of the code files :)

**Comment by thygate** [comment]
You can edit `ui-config.json` in the webui root, and change these lines, change from true to false :
`"customscript/depthmap.py/txt2img/Combine into one image./value": false,`
and
`"customscript/depthmap.py/img2img/Combine into one image./value": false,`

I've set it to default False in next version.

**Comment by SebSaga** [comment]
hmmm I gave it a shot but it didn't seem to work for me. I'm also a complete noob so its's very possible that I did something wrong haha!

I can definitely wait for the next version though, I appreciate all the effort you put into this!

**Reply by thygate** [reply]
I just made the commit.

**Reply by SebSaga** [reply]
Amazing!!! Just updated, I especially love the ability to remove the background. Awesome stuff thygate, thank you so much for your contributions to this.

**Reply by thygate** [reply]
Thank [@graemeniedermayer](https://github.com/graemeniedermayer) for the PR =)
